src/app/item-list/item-list.component.html

```diff
@@ -1,6 +1,6 @@
-<h2 id="title"> {{capitalizeFirstLetter(this.url[1])}}</h2>
-  <div class="container">
-    <div class="row card-deck">
+<h2 id="title" *ngIf="!selectedStore"> {{capitalizeFirstLetter(this.url[1])}}</h2>
```

Comment:
You can use a pipe instead of method. Have a look at: 
https://gist.github.com/chl03ks/c7fbd8b7f632f5891e98854b024f0557